build/tmpl/bra.toml

```diff
@@ -0,0 +1,19 @@
+# default configuration created by the `mage watch` command.
+# this file can be edited and should be checked into source control.
+# see https://github.com/unknwon/bra/blob/master/templates/default.bra.toml for more configuration options.
+[run]
+init_cmds = [
+  ["mage", "-v", "build:backend"],
+  ["mage", "-v" , "reloadPlugin"]
+]
+watch_all = true
+follow_symlinks = false
+watch_dirs = [
+  "pkg",
```

Comment:
Would be nice to reload when plugin.json is changed as well, but maybe leave that for later?

Reply:
sounds good. i'll add that 👍 

thanks for the tip, i wasn't aware that the backend was dependent on `plugin.json`. how is it used?

Reply:
added by dc48129619c194fcc918abe9b204bf3f96c6a12e. it doesn't look like it's possible to specify a single file, so i configured it to watch for changes to `.go` or `.json` files in `src` or `pkg`